crates/storage/codecs/derive/src/compact/generator.rs

```diff
@@ -1,11 +1,13 @@
-//! Code generator for the `Compact` trait.
+//! Code generator for the Compact trait.
```

Comment:
please revert all of these doc changes, especially the ones that link to structs

Reply:
I have already taken care of most of the issues brought up by @mattsse , I just haven't pushed yet cause on the last issue, I need clarifications on that @onbjerg